host_core/native/hostcore_wasmcloud_native/src/par.rs

```diff
@@ -40,18 +40,18 @@ pub(crate) fn extract_target_bytes(par: &ProviderArchive) -> Result<Vec<u8>, Err
 }
 
 pub(crate) fn cache_path(
-    subject: String,
-    rev: u32,
-    contract_id: String,
-    link_name: String,
+    subject: &str,
+    rev: i32,
```

Comment:
I think this should still be `u32` unless there's a reason I'm missing to support negative numbers (feels weird for filepath)

Reply:
yeah revisions are never negative.

Reply:
I had actually changed it, but the actual claims type from wascap is an `i32`, so it didn't really make sense to change it. I just changed it to match what things were set to